packages/p2p/src/components/modal-manager/modals/filter-modal/filter-modal-header.tsx

```diff
@@ -25,7 +29,6 @@ const FilterModalHeader = ({ pageHeaderReturnFn }) => {
                             className='filter-modal__header-return-button'
```

Comment:
Please rename the classnames to use the component name.
```suggestion
                            className='filter-modal-header__return-button'
```